app/views/works/_standard_form.html.erb

```diff
@@ -265,6 +265,7 @@
         <span class="html-notes"><%= allowed_html_instructions %></span>
         <span class="hidden rtf-notes"><%= ts('Type or paste formatted text.') %><%= link_to_help "rte-help" %></span>
       </p>
+      <p class="notice"><%= ts("<strong>Note:</strong> Text entered in the posting form is <strong>not</strong> automatically saved. Always keep a backup copy of your work.</p>")%>
```

Comment:
I found this worth a read.

http://yehudakatz.com/2010/02/01/safebuffers-and-rails-3-0/